napari/viewer.py

```diff
@@ -233,3 +268,26 @@ def current_viewer() -> Optional[Viewer]:
         return None
     else:
         return _QtMainWindow.current_viewer()
+
+
+viewer = napari.Viewer()
+
+
+def get_center_bbox(shape):
+    """Get the center coordinate, height, width of the shape roi
+
+    Parameters
+    ----------
+    shapes : napari.layers.shape
+           A napari shapes layer
+
+    Returns
+    -------
+    center coords, height and width of shape: float
+           The center coordinates, height and widht of shape roi
+    """
+    height, width = shape.max(axis=0) - shape.min(axis=0)
+    min_y, min_x = shape.min(axis=0)
+    center_coords = [min_y + height / 2, min_x + width / 2]
+
+    return center_coords, height, width
```

Comment:
Some leftover code I guess :)

Reply:
This is the function that gets the center coordinate for each shapes ROI before a screenshot is taken and stored in a list. It's legit code.

Reply:
It's duplicated code (the real one is in `geometry.py` right?), and unused as far as I can tell.

Reply:
You are right! And I will clean it up. I didn't remember adding it to geometry.py and calling it in qt_main_windows.py